editor/effects.js

```diff
@@ -280,6 +277,12 @@ export default {
 
 		return effects;
 	},
+	INITIALIZE_META_BOX_STATE( action ) {
+		// Hold jquery.ready until the metaboxes load
+		if ( action.metaBoxes.side || action.metaBoxes.normal ) {
```

Comment:
Do we need to account for `action.metaBoxes.advanced` ? I guess we're not showing these?

Otherwise this could work well:

```js
if ( some( action.metaBoxes ) ) {
```

Reply:
I found that "advanced" and "normal" are the showing the same metaboxes somehow. We're still not showing the metaboxes supposed to appear under the title but I'm not sure how these are registered.